src/multiselectMixin.js

```diff
@@ -709,6 +719,45 @@ export default {
         this.preferredOpenDirection = 'above'
         this.optimizedHeight = Math.min(spaceAbove - 40, this.maxHeight)
       }
+    },
+    /**
+     * Tried to find option based on input event value
+     * and select it. If option was find - closes dropdown.
+     * Otherwise just update search value.
+     */
```

Comment:
```suggestion
    /**
     * Try to find option based on input event value
     * and select it. If option is found - close dropdown.
     * Otherwise just update the search value.
     */
```